src/CommandLineParser.ts

```diff
@@ -256,3 +257,34 @@ export function createDiagnosticReporter(pretty: boolean, system = ts.sys): ts.D
         reporter(diagnostic);
     };
 }
+
+export function locateConfigFile(commandLine: ParsedCommandLine): ts.Diagnostic | string | undefined {
```

Comment:
As discussed on discord, I don't think this belongs in here, especially since it's not even called form this file. I would like to either keep it close to where it is called, or maybe in a CLIHelpers.ts file or something similar.